m4/odp_pthread.m4

```diff
@@ -1,7 +1,15 @@
 # ODP_PTHREAD
 # -----------
 # Check for pthreads availability
-AC_DEFUN([ODP_PTHREAD], [dnl
-AX_PTHREAD([CC="$PTHREAD_CC"],
-	   [AC_MSG_FAILURE([We require pthreads to be available])])
+AC_DEFUN([ODP_PTHREAD], [
+	AC_MSG_CHECKING([for pthread support in -pthread])
+	AC_LANG_PUSH([C])
+	CFLAGS="$CFLAGS -pthread"
+	LDFLAGS="$LDFLAGS -pthread"
```

Comment:
I've mixed feelings towards this change. Why do you want to drop PTHREAD_CFLAGS/PTHREAD_LIBS? Several should have been compiled w/o pthreads, IIRC (proc mode ones).